kpatch-build/lookup.c

```diff
@@ -92,9 +91,9 @@ static int maybe_discarded_sym(const char *name)
 }
 
 static int locals_match(struct lookup_table *table, int idx,
```

Comment:
While you're at it, can you change locals_match() and maybe_discarded_sym() to return bool?  Their int return codes are opposite than normal kernel error code expectations, making them extra confusing.  Might as well clean those up while we're touching the code.

Reply:
Ok